frontend/app/globals.css

```diff
@@ -1,3 +1,6 @@
+/* Remove this line */
+/* @import "../styles/buttons.css"; */
```

Comment:
Couldn't you import it on this file directly? (but after the @tailwind commands)